wine-staging/PKGBUILD

```diff
@@ -0,0 +1,172 @@
+            'bdde7ae015d8a98ba55e84b86dc05aca1d4f8de85be7e4bd6187054bfe4ac83b5a20538945b63fb073caab78022141e9545685e4e3698c97ff173cf30859e285'
+            '446b3d4ce7ed7d3cdd395e97ccc773a044cd848a107045e84d02d3f33473a0214d5f64179a301fb953aebe702b92eff0f2a614eca4f968bba064916bb8ca5624')
+validpgpkeys=(5AC1A08B03BD7A313E0A955AF5E6E9EEB9461DD7
+              DA23579A74D4AD9AF9D3F945CEFAC8EAAF17519D)
+
+pkgdesc="A compatibility layer for running Windows programs - Staging branch"
+url="https://www.wine-staging.com"
+arch=(x86_64)
+options=(staticlibs !lto)
+license=(LGPL-2.1-or-later)
+depends=(
+  attr            #lib32-attr
+  desktop-file-utils
+  fontconfig      #lib32-fontconfig
+  freetype2       #lib32-freetype2
+  gcc-libs        #lib32-gcc-libs
+  gettext         #lib32-gettext
+  libpcap         #lib32-libpcap
+  libxcursor      #lib32-libxcursor
+  libxi           #lib32-libxi
+  libxrandr       #lib32-libxrandr
+)
+makedepends=(autoconf bison perl flex mingw-w64-gcc
+  git
+  alsa-lib              #lib32-alsa-lib
+  ffmpeg
+  giflib                #lib32-giflib
+  gnutls                #lib32-gnutls
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  gtk3                  #lib32-gtk3
+  libcups               #lib32-libcups
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libva                 #lib32-libva
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  libxxf86vm            #lib32-libxxf86vm
+  mesa                  #lib32-mesa
+  mesa-libgl            #lib32-mesa-libgl
+  opencl-headers
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  python
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  v4l-utils             #lib32-v4l-utils
+  vulkan-icd-loader     #lib32-vulkan-icd-loader
+)
+makedepends+=(
+  ntsync-header
+)
+optdepends=(
+  alsa-lib              #lib32-alsa-lib
+  alsa-plugins          #lib32-alsa-plugins
+  cups                  #lib32-libcups
+  dosbox
+  ffmpeg
+  giflib                #lib32-giflib
+  gnutls                #lib32-gnutls
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  gtk3                  #lib32-gtk3
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libva                 #lib32-libva
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  v4l-utils             #lib32-v4l-utils
+  vulkan-icd-loader     #lib32-vulkan-icd-loader
+  wine-gecko
+  wine-mono
+)
+optdepends+=(
+  NTSYNC-MODULE
+)
+
+provides=("wine=$pkgver")
+conflicts=('wine')
+install=wine.install
+
+prepare() {
+  # Get rid of old build dirs
+  rm -rf $pkgname-{32,64}-build
+  mkdir $pkgname-{32,64}-build
+
+  cd wine
+  # apply wine-staging patchset
+  ../wine-staging/staging/patchinstall.py --backend=git-apply --all \
+    --no-autoconf \
+    -W eventfd_synchronization \
+    -W server-PeekMessage \
```

Comment:
From my experience, `server-PeekMessage` is fine and it should not be excluded when ntsync is used.  But I could be wrong, maybe you had a different motivation for that.

Reply:
For me it fails to apply the ntsync patch, that's why I kept it out.

Reply:
All right, seems to be I just used to rebase ntsync over staging patchset, so it was not a problem for me. Let's just leave it at that.